File: Exavision.Seasense.Shared/Settings/SettingCamera.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Exavision.Seasense.Shared.Settings {
    public class SettingCamera : SettingMaterial {
        public string StreamUrl { get; set; }

        public int StreamWidth { get; set; }

        public int StreamHeight { get; set; }
    }
}

File: Exavision.Seasense/Client/src/app/materials/settings/setting-camera.ts
import { MaterialType } from "../material-type";
import { SettingCapability } from "./setting-capability";
import { SettingMaterial } from "./setting-material";


export interface SettingCamera extends SettingMaterial {
  streamUrl: string;
  streamWidth: number;
  streamHeight: number;
}


